src/gt4py/next/ffront/decorator.py

```diff
@@ -65,9 +67,21 @@
     sym,
 )
 from gt4py.next.program_processors import processor_interface as ppi
+from gt4py.next.program_processors.runners.dace_iterator import workflow as dace_workflow
 from gt4py.next.type_system import type_info, type_specifications as ts, type_translation
 
 
+try:
+    import dace
+    from dace.frontend.python.common import SDFGConvertible
+
+    from gt4py.next.program_processors.runners.dace import run_dace_gpu
+except ImportError:
+    dace: Optional[ModuleType] = None  # type:ignore[no-redef]
+
+    class SDFGConvertible: ...
```

Comment:
This line (`class SDFGConvertible: ...`) is probably not needed anymore after the latest change.

Reply:
good catch